docs/Fields.md

```diff
@@ -894,6 +894,14 @@ PriceField.defaultProps = {
 ```
 {% endraw %}
 
+
+## Third-Party Components
```

Comment:
Please add it to the Ecosystem documentation instead

Reply:
Ok. I moved all third-part links to Ecosystem, to be consistent.
This could be part of this PR or another one, your choice.

Reply:
So your last changes makes me realize that what I asked you is wrong, sorry. The third party field should be added to the Fields doc, and the Ecosystem should be updated to add a link to the third party fields.